ReplayKitExample/ReplayKitExample/PresentationViewController.swift

```diff
@@ -0,0 +1,641 @@
+//
+//  PresentationViewController.swift
+//  ReplayKitExample
+//
+//  Copyright © 2020 Twilio. All rights reserved.
+//
+
+import UIKit
+import TwilioVideo
+
+enum DataSourceError: Error {
+    // The room is not connected.
+    case notConnected
+}
+
+class PresentationViewController : UIViewController {
```

Comment:
The Viewer UI uses iOS 13 only features: 

1. SF Symbols for mic / video / disconnect icons
2. UIContextMenuConfiguration to present press and hold action menus

Perhaps the viewer UI could be made iOS 13 only.

Reply:
I guess its okay to have viewer UI only in iOS 13.